core/common/src/EntityProps.ts

```diff
@@ -83,69 +84,113 @@ export enum PrimitiveTypeCode {
   Point2d = 0x701, // eslint-disable-line @typescript-eslint/no-shadow
   Point3d = 0x801, // eslint-disable-line @typescript-eslint/no-shadow
   String = 0x901,
-  IGeometry = 0xa01, // Used for Bentley.Geometry.Common.IGeometry types
+  IGeometry = 0xa01,
 }
 
-/** A callback function to process properties of an Entity
- * @beta
+/** A callback function used when iterating over the properties of an [Entity]($backend) class using methods like
+ * [Entity.forEachProperty]($backend) and [IModelDb.forEachMetaData]($backend).
+ * @public
  */
 export type PropertyCallback = (name: string, meta: PropertyMetaData) => void;
 
-/** A custom attribute instance
- * @beta
+/** Represents a [custom attribute]($docs/bis/ec/ec-custom-attributes.md) attached to a [[PropertyMetaData]] or [[EntityMetaData]].
+ * @public
  */
 export interface CustomAttribute {
-  /** The class of the CustomAttribute */
+  /** The fully-qualified name of the [custom attribute class]($docs/bis/ec/ec-custom-attribute-class.md).*/
   ecclass: string;
   /** An object whose properties correspond by name to the properties of this custom attribute instance. */
   properties: { [propName: string]: any };
 }
 
 type FactoryFunc = (jsonObj: any) => any;
 
-/** @beta */
+/** JSON representation of a [[PropertyMetaData]].
+ * @public
+ */
 export interface PropertyMetaDataProps {
+  /** See [[PropertyMetaData.primitiveType]]. */
   primitiveType?: number;
+  /** See [[PropertyMetaData.structName]]. */
   structName?: string;
+  /** See [[PropertyMetaData.extendedType]]. */
   extendedType?: string;
+  /** See [[PropertyMetaData.description]]. */
   description?: string;
+  /** See [[PropertyMetaData.displayLabel]]. */
   displayLabel?: string;
+  /** See [[PropertyMetaData.minimumValue]]. */
   minimumValue?: any;
+  /** See [[PropertyMetaData.maximumValue]]. */
   maximumValue?: any;
+  /** See [[PropertyMetaData.minimumLength]]. */
   minimumLength?: number;
+  /** See [[PropertyMetaData.maximumLength]]. */
   maximumLength?: number;
+  /** See [[PropertyMetaData.readOnly]]. */
   readOnly?: boolean;
+  /** See [[PropertyMetaData.kindOfQuantity]]. */
   kindOfQuantity?: string;
+  /** See [[PropertyMetaData.isCustomHandled]]. */
   isCustomHandled?: boolean;
+  /** See [[PropertyMetaData.isCustomHandledOrphan]]. */
```

Comment:
This is supposed to be determined by `AutoHandledPropertiesCollection::DetectOrphanCustomHandlerProperty`. Nobody ever calls that function. @khanaffan did I overlook something or is this property indeed meaningless and never true?

Reply:
No, you are right. From what i see `s_orphanCustomHandledProperties` will always be empty as `DetectOrphanCustomHandledProperty` is never called from any where.

In addition even if its called the result will be exactly same is `isCustomHandled`. I am not sure why its called "Orphan" when it also check for exactly same customAttribute  presence on the property as the isCustomHandled does.

As of now if you remove this it be fine as it will always be `false`.